addons/account/models/partner.py

```diff
@@ -10,11 +10,70 @@
 
 from odoo import api, fields, models, _
 from odoo.osv import expression
-from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT, mute_logger
+from odoo.tools import mute_logger, LazyTranslate
 from odoo.exceptions import ValidationError, UserError
 from odoo.addons.base.models.res_partner import WARNING_MESSAGE, WARNING_HELP
 from odoo.tools import SQL, unique
-from odoo.addons.base_vat.models.res_partner import _ref_vat
+
+_lt = LazyTranslate(__name__)
+
+_ref_vat = {
+    'al': 'ALJ91402501L',
+    'ar': _lt('AR200-5536168-2 or 20055361682'),
```

Comment:
Eh, this also means that we need to change the module of translations...

Might be too much of work for something that should be working in the end

Reply:
Okayyy I think it's okay as well :smile: 
Closing then!